CCDScan documentation (#513)
Concordium/concordium.github.io: source/mainnet/net/resources/ccd-scan-transactions.rst

.. image:: ../images/ccd-scan/ccd-scan-home-transaction-success.png

|

The transaction details shows the following.

- **Transaction**: the first six characters of the transaction hash. Click Copy |copy| to copy the entire transaction hash. It also shows the transaction state.
- **Block height/block hash**: the block height and the first six characters of the block hash. Click Copy |copy| to copy the entire block hash. Click the block hash to see the :ref:`account details<blocks-view>`.
- **Age**: the transaction age with date/time stamp.
- **Transaction type/cost**: the transaction type and the transaction fee.
- **Sender**: the first six characters of the account that sent the transaction if available for that transaction. Click Copy |copy| to copy the entire hash. Click the account hash to see the :ref:`account details<accounts-view>`.
- **Events/Reject reason**:  list of events included in the transaction. It contains more details about the transaction, including drill-through links and copy options where available. There are many different transaction types and descriptions.

> A successful transaction shows the **Events** that were included in the transaction.

Reviewer: ```suggestion
A successful transaction shows the **Events** that were the result of the transaction.
```

> - **Events/Reject reason**:  list of events included in the transaction. It contains more details about the transaction, including drill-through links and copy options where available. There are many different transaction types and descriptions. If you see the transaction type **Update micro CCD per Euro**, it is possible to see the new rate and the effective timestamp.

Reviewer: Is "Update micro CCD per Euro" treated in a special way? There are other chain updates, are those not displayed in detail?

If the transaction was rejected, you see something similar to the following:

.. image:: ../images/ccd-scan/ccd-scan-home-transaction-reject.png

|

> The only difference between a successful and rejected transaction is that the rejected transaction shows the **Reject reason** instead of the events.

Reviewer: Not sure what the context for "the only difference" is.

But I would not phrase it like this. Instead I would say something like

"The only effect of a rejected transaction is the payment for execution cost. In contrast, a successful transaction affects the state of the chain. The effects are described by the generated events."
